src/StimulusBundle/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.x
+
+-   Added Typescript controllers support
```

Comment:
I think there should be doc somewhere explaining a bit more this change.. as it could be disappointing for some :)

Reply:
Good point - just a bit more clarity on how `.ts` controller files are now located (if you use TypeScript bundle to process in the first place)

Reply:
Yeah fair enough, I'll try to add some documentation